### DISCUSSION TITLE
希望可以通过环境变量，设置开启某些插件、并设置插件token

### DISCUSSION THREAD

**Discussion by dupl** [Dogtiti/ChatGPT-Next-Web-EarlyBird#8] [General]
希望可以通过环境变量，设置开启某些插件、并设置插件token。插件仓库：https://github.com/ChatGPTNextWeb/NextChat-Awesome-Plugins

**Comment by Dogtiti** [comment]
并不是开启了插件就能用，要看模型是否支持插件以及目前对该模型是否做了插件的适配
